index.html

```diff
@@ -151,6 +244,115 @@ <h2 class="works-2-div2-1-a-p">Multi-Post<br>Stories</h2>
                     </button>
                 </div>
             </div>
+        </div>
+                <!--works2-->
+        <div class="works-2">
+            <div class="works-2-div1">
+                <img class="works-2-div1-img" src="images/Snapshoot Portfolio2.svg" alt="">
+                <img class="works-2-div1-img-desktop" src="images/Snapshoot Portfoliowork2.svg" alt="">
+            </div>
+            <div class="works-2-div2">
+                <div class="works-2-div2-1">
+                    <div class="works-2-div2-1-a">
+                        <h2 class="works-2-div2-1-a-p">Multi-Post<br>Stories</h2>
+                    </div>
+                    <div class="works-2-div2-1-b">
+                        <ul class="works-2-div2-1-b-ul">
+                            <li class="works-2-div2-1-b-ul-li-1">
+                                <span class="works-2-div2-1-b-ul-li-1-sp">CANOPY</span>
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-2">
+                                <img class="works-2-div2-1-b-ul-li-2-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-3">
+                                <span class="works-2-div2-1-b-ul-li-3-sp">Back End Dev</span>
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-4">
```

Comment:
- [ ] Good job adding a proper project item on your web page, :clap: , I’ve noticed that there is some overflow with the `works-2` text , where you are working on screen that have `768px` , Kindly make sure you fix it :+1: , it will make your web page more responsive :+1: 


[Screencast from 02-25-2023 12_05_32 AM.webm](https://user-images.githubusercontent.com/80458199/221303585-a710e4b7-16c0-4a0c-9a9f-11e20cb81df9.webm)

Reply:
> * [ ]  Good job adding a proper project item on your web page, 👏 , I’ve noticed that there is some overflow with the `works-2` text , where you are working on screen that have `768px` , Kindly make sure you fix it 👍 , it will make your web page more responsive 👍

@iambenkis  the units given to me only works for desktop with width 1440 and above